release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1560,6 +1560,8 @@ sourceStrategy:
 
 * Previously, a rare bug caused inconsistency during AWS cluster installations. The installation program has been updated to avoid this scenario. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2046277[*BZ#2046277*])
 
+* * Previously, the ratio of user defined tags to the reserved {product-title} tags was 8:2 for AWS resources. With this release, the new ratio is 25:25. You can now add upto 25 user tags during installation.   
```

Comment:
I think, "number of supported" would make more sense than the "ratio" terminology.